pints/_plot.py

```diff
@@ -0,0 +1,248 @@
+#
+# Quick diagnostic tools for inference results
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+import pints
+import numpy as np
+from scipy import stats
+#import matplotlib.pyplot as plt
+import pylab as plt
+
+def trace(chain):
```

Comment:
Could you change the signature of this method to `trace(chain, *args)` and make it plot one or multiple chains? (The first argument is to ensure there's at least one chain)

https://stackoverflow.com/questions/3394835/args-and-kwargs#3394898

Reply:
Done